### PR TITLE
feat(spans): Scrub span descriptions on resource-like operations

### PR DESCRIPTION
**Pull request by iker-barriocanal**
Scrub span descriptions on `resource*` operations.

#skip-changelog

### CODE REVIEW

**Review by jjbayer — APPROVED** [review]
Same comments as on https://github.com/getsentry/relay/pull/2127:

* else if
* negative tests

File: relay-general/src/store/regexes.rs [hunk 1]
```diff
@@ -88,3 +88,31 @@ pub static CACHE_NORMALIZER_REGEX: Lazy<Regex> = Lazy::new(|| {
...
 });
+
+/// Regex with multiple capture groups for resource tokens we should scrub.
+///
+/// Resource tokens are the tokens that exist in resource spans that generate
+/// high cardinality or are noise for the product. For example, the hash of the
+/// file next to its name.
+///
+/// Slightly modified Regex from
+/// https://github.com/getsentry/sentry/blob/de5949a9a313d7ef0bf0685f84fe6e981ac38558/src/sentry/utils/performance_issues/base.py#L292-L306
```
Comment (olksdr) [inline relay-general/src/store/regexes.rs]:
```suggestion
/// <https://github.com/getsentry/sentry/blob/de5949a9a313d7ef0bf0685f84fe6e981ac38558/src/sentry/utils/performance_issues/base.py#L292-L306>
```

**Review by olksdr — APPROVED** [review]
same comments as @jjbayer

**Review by olksdr — APPROVED** [review]
same comments as @jjbayer

**Comment by iker-barriocanal** [comment]
@jjbayer what do you mean with "else if"?
Added a negative test in [c5f0b0e](https://github.com/getsentry/relay/pull/2129/commits/c5f0b0e94fc822b11f6752870157bb5d652a958d).

**Pull request merged**